idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -393,6 +393,23 @@ service WorkflowService {
       4: shared.LimitExceededError limitExceededError,
     )
 
+  /**
+  * Reset the sticky tasklist related information in mutable state of a given workflow.
+  * Things cleared are:
+  * 1. StickyTaskList
+  * 2. StickyScheduleToStartTimeout
+  * 3. ClientLibraryVersion
+  * 4. ClientFeatureVersion
+  * 5. ClientImpl
+  **/
+  shared.ResetStickyTaskListResponse ResetStickyTaskList(1: shared.ResetStickyTaskListRequest resetRequest)
```

Comment:
Why we need to return empty response here? Is this Reset API going to expect anything in future?
Otherwise shouldn't we just return void instead of an empty response?

Reply:
Yes, it seems in Wenquan's original code it is for future return values.

struct ResetStickyTaskListResponse {
  // The reason to keep this response is to allow returning
  // information in the future.
}